src/common/database/mongoConfig.ts

```diff
@@ -0,0 +1,33 @@
+import { MongoClient, ServerApiVersion } from 'mongodb';
+
+/* import { getMongoAtlasUri } from '@common/utils/envConfig'; */
+import { logger } from '@src/server';
+
+const uri =
+  'mongodb+srv://robertvilladev:30x3EuPvXrEpxByi@database.vjpjaep.mongodb.net/?retryWrites=true&w=majority&appName=database';
```

Comment:
hello database credentials, you should probably remove this & rotate your credentials.
![image](https://github.com/robertvilladev/node-express-typescript-demo/assets/10256477/635c12d4-7a34-4a18-bc72-0214a5b70287)

Reply:
ohh thanks! im using .env variables but I forgot to change it here jaja !